xgboost-demo-all.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
Can we simply name it xgboost-demo or xgboost-integration?

xgboost-demo-all seems a bit weird.

Reply:
Sure